crates/ruff_python_stdlib/src/identifiers.rs

```diff
@@ -26,8 +33,9 @@ pub fn is_mangled_private(id: &str) -> bool {
 /// Returns `true` if a string is a PEP 8-compliant module name (i.e., consists of lowercase
 /// letters, numbers, and underscores).
 pub fn is_module_name(s: &str) -> bool {
-    s.chars()
-        .all(|c| c.is_lowercase() || c.is_numeric() || c == '_')
+    is_identifier(s)
+        && s.chars()
+            .all(|c| c.is_lowercase() || c.is_numeric() || c == '_')
```

Comment:
NIT: It could make sense to duplicate the `is_identifier` logic to only iterate two times over the string, instead of 3 times (at least once to check if it is in the KWARGS list, once to test if it is an identifier, and once to test if it is a module)